### PR TITLE
Update infix to postfix

### PR DESCRIPTION
**Pull request by shellhub**
### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: data_structures/stacks/__init__.py [hunk 1]
```diff
@@ -1,22 +0,0 @@
-class Stack:
```
Comment (poyea) [inline data_structures/stacks/__init__.py:1]: Not sure if this is okay (any other files are using this?).

Comment (shellhub) [reply data_structures/stacks/__init__.py:1]: This file should be empty. No other files are using this

Comment (cclauss) [reply data_structures/stacks/__init__.py:1]: Do the other files all have tests so that we know that is true?

Comment (shellhub) [reply data_structures/stacks/__init__.py:1]: Anything need to be updated ?

Comment (poyea) [reply data_structures/stacks/__init__.py:1]: seems like there's another stack inside stack/

Comment (shellhub) [reply data_structures/stacks/__init__.py:1]: Building is passed. So I think we should clear content in __init__.py

**Comment by shellhub** [comment]
@cclauss @poyea This PR can be merged now ?

**Review by poyea — APPROVED** [review]

**Pull request merged**